Kitodo/src/main/java/org/kitodo/production/services/data/ImportService.java

```diff
@@ -11,45 +11,33 @@
 
 package org.kitodo.production.services.data;
 
-import java.io.File;
-import java.io.FileOutputStream;
+import com.sun.jersey.api.NotFoundException;
```

Comment:
An other `NotFoundException` class? One from Jersey and the previous one from Spring.